index.html

```diff
@@ -1,67 +1,47 @@
 <!DOCTYPE html>
 <html>
-<head>
-<link rel="stylesheet" href="./styles.css">
-<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
-<script type="text/javascript" src="main.js"></script>
-  <title>What's for dinner?</title>
+  <head>
+   <meta charset="UTF-8">
+    <link rel="stylesheet" href="./styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>What's for dinner?</title>
+  </head>
   <body>
-    <nav>
+      <section class="top-border">
+      <nav class="top-nav">
       <header class="page-title">
       <h1> What's for Dinner?</h1>
       <button class="add-recipe"> Add a recipe </button>
-    </nav>
-    <main>
-      <section>
-      <div class="form-view">
+      </nav>
+      <main>
+    <section class="recipe-form">
+      <section class="form-view">
         <form id="form">
-        <p id="form-prompt"> What are you looking for?</p>
-          <div>
-            <span>
+        <div>
+        <p id="form-prompt"> What are you looking for?<span>*</span></p>
             <ul><input type="radio" id="side-option"
             name="food" value="side-recipe">
             <label for="side-option">Side</label></ul>
-          </div>
-        </span>
-        <div>
-          <span>
             <ul><input type ="radio" id="main-option"
             name="food" value="main-recipe">
             <label for="main-option">Main</label></ul>
-          </div>
-            </span>
-            <div>
-              <span>
             <ul><input type="radio" id="dessert-option"
             name="food" value="dessert-recipe">
             <label for="dessert-option">Dessert</label></ul>
-          </div>
-            </span>
-            <div>
-              <span><ul><input type="radio" id="entire-meal-option"
+            <ul><input type="radio" id="entire-meal-option"
             name="food" value="entire-meal-recipe">
             <label for="entire-meal-option">Entire Meal</label></ul>
             <div>
             <button type="meal-time">Let's Cook</button>
-          </div>
-          </div>
-        </span>
-          </div>
-        </section>
-          <!-- <div>
-          <button type="meal-time">Let's Cook</button>
-        </div> -->
+           </div>
         </form>
-        <pre id="log-meal">
-        </pre>
-        <section>
-      <div class="image-cookpot">
-      <img src= "assets/cookpot.svg" alt="image of a cooking pot"/>
-     </div>
-   </sectiom>
-    </main>
+        <section id="log-meal">
```

Comment:
hint: these `<section>` tags will probably  need closing tags `</section>` when you get to them!

Reply:
Thank you!!